packages/react-dom/npm/ReactDOMDispatcher.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+function checkDCE() {
```

Comment:
We should remove this btw, it's just for the main entry point.

Reply:
will do, thanks!